core-java-modules/pom.xml

```diff
@@ -129,6 +129,7 @@
         <module>core-java-regex-2</module>
         <module>core-java-uuid</module>
         <module>pre-jpms</module>
+        <module>core-java-string-apis-2</module>
```

Comment:
Note that the modules are arranged alphabetically, so we should move this one up, after the core-java-string-apis module.

Reply:
Updated @KevinGilmore .
Thanks

Reply:
Updated @KevinGilmore .
Thanks!